front/src/modules/study/components/AddOrEditStudyModal.tsx

```diff
@@ -256,15 +256,16 @@ export default function AddOrEditStudyModal({ editionMode, study }: Props) {
                       </div>
                     }
                     value={{
-                      id: currentStudy.study_type,
+                      id: currentStudy.study_type || 'nothingSelected',
                       label: t(
                         `studyCategories.${currentStudy.study_type || 'nothingSelected'}`
                       ).toString(),
                     }}
                     options={studyCategoriesOptions}
-                    onChange={(e) =>
-                      handleStudyInputChange('study_type', e?.id as StudyForm['study_type'])
-                    }
+                    onChange={(e) => {
+                      const selectedType = e?.id === 'nothingSelected' ? '' : e?.id;
```

Comment:
`e` is never undefined, but `id` can be undefined. It's better to test in this order:
- if `e.id` is not undefined and not equal to `'nothingSelected'`, then use `e.id`
- else return `''`

Reply:
With the new behavior, e.id equals either the studyType or an empty string (for the nothingSelected case) so I think we can totally remove this condition.